applications/isce2geotiff.py

```diff
@@ -92,29 +113,71 @@ def get_cmap(mapname, N, clim):
     ####Set up the color table
     if inps.table is None: ####No custom color map has been provided
         cmap = get_cmap(inps.cmap, inps.ncolors, inps.clim)
+        print(cmap)
         plt_cmap = True
     else:
         cmap = table
         plt_cmap = False
 
+    #####Build Wrapped VRT
+    outvrtname = inps.outfile+'.vrt'
+    if inps.wrap:
+        vrtname = inps.infile+'.vrt'
+        invrtname = inps.outfile+'.phs.tif'
+        tifoutname = inps.outfile+'.wrapped.tif'
+        cmd = 'gdal_calc.py --type Float32 -A {0} --calc="numpy.angle(A)" --NoDataValue=0.0  --outfile={1} --overwrite'.format(vrtname, invrtname)
+        #gdal_calc.Calc('numpy.angle(A)', A=vrtname, outfile = invrtname,NoDataValue=0, overwrite=True)
+        print(cmd)
+        
+        flag = os.system(cmd)
 
-    #####Build VRT
-    vrtname = inps.outfile+'.vrt'
-    if os.path.exists(vrtname):
-        print('VRT file already exists. Cleaning it ....')
-        os.remove(vrtname)
+        if flag:
+           raise Exception('Failed: %s'%(cmd))
 
-    cmd = 'gdaldem color-relief {0} {1} {2} -alpha -b {3} -of VRT'.format(inps.infile, cmap, vrtname, inps.band+1)
-    
-    flag = os.system(cmd)
-    if flag:
-        raise Exception('Failed: %s'%(cmd))
+    else:
+    #####Build Unwrapped VRT 
+        invrtname = inps.infile+'.vrt'
+        tifoutname = inps.outfile+'.unwrapped.tif'
+        if os.path.exists(outvrtname):
+           print('VRT file already exists. Cleaning it ....')
+           os.remove(outvrtname)
+
+    ###Build geotiff
+    if os.path.exists(tifoutname):
+        print('TIF file already exists. Cleaning it ....')
+        os.remove(tifoutname)
+
+    outvrtname = gdal.DEMProcessing('',invrtname,'color-relief', colorFilename = cmap, format = 'MEM', addAlpha =True, options = ['-b', str(inps.band+1), '-of','VRT']) 
 
     ###Build geotiff
-    cmd = 'gdal_translate {0} {1}'.format(vrtname, inps.outfile)
+    if os.path.exists(tifoutname):
+        print('TIF file already exists. Cleaning it ....')
+        os.remove(tifoutname)
+    print('Creating TIF file:', tifoutname)
+    ds = gdal.Translate(tifoutname,outvrtname)
+    ds = None	
+    
+
+   ####Use water mask to Crop RGB tiff
+    if inps.mask is None:
+        mask = None 
+    else:
+        maskinput = inps.mask
+        cmd = 'gdal_rasterize -b 1 -b 2 -b 3 -b 4 -burn 0 -burn 0 -burn 0 -burn 0 {0} {1}'.format(maskinput,tifoutname)
```

Comment:
This can also be pythonized. 
https://gis.stackexchange.com/questions/212795/rasterizing-shapefiles-with-gdal-and-python